adage/analyze/views.py

```diff
@@ -1,22 +1,42 @@
-from rest_framework.decorators import api_view
-from rest_framework.response import Response
-from rest_framework.reverse import reverse
 from rest_framework.viewsets import ReadOnlyModelViewSet
+from .models import Experiment, MLModel
+from .serializers import ExperimentSerializer, MLModelSerializer
 
-from .models import MLModel
-from .serializers import MLModelSerializer
 
+class ExperimentViewSet(ReadOnlyModelViewSet):
+    """Experiments viewset"""
+    http_method_names = ['get']
+    serializer_class = ExperimentSerializer
+
+    def get_queryset(self):
+        queryset = Experiment.objects.all()
+        # 'search' parameter, which does full text search based on the
+        # following 3 fields in Experiment model:
+        # - "accession"
+        # - "name"
+        # - "description"
+        search_str = self.request.query_params.get('search', None)
+        if search_str is not None:
+            from django.contrib.postgres.search import (
+                SearchQuery, SearchRank, SearchVector
+            )
+            # In AWS Postgres RDS, the default config is 'simple'.
+            # We use 'english' to make the text search more flexible.
```

Comment:
This sounds like a good change, but I am concerned about an AWS dependency getting lodged this deep into our code. Is there a way to specify the default we want in a config file instead?

Reply:
This is another Postgres-specific feature. Technically speaking, it is not really AWS-dependent. It just makes `english` config explicit.

Reply:
Ok, let's remove the AWS reference from the comment if the reason for overriding the default is not specific to AWS. I still think it would be preferable to use a single centralized config setting, if possible, to enforce this rather than overriding it on each call. It would make for cleaner and more maintainable code.